src/backend/tests/teams.test.ts

```diff
@@ -273,18 +264,6 @@ describe('Teams', () => {
       await expect(callSetTeamLeads).rejects.toThrow(expectedException);
     });
 
-    test('setTeamLeads lead is a member', async () => {
```

Comment:
Why did you remove this test? Could it be updated to just expect the correct response instead?

Reply:
Sorry, that's what I meant to do, but I think I must've accidentally removed it. I'll fix that real quick too.

Reply:
> Why did you remove this test? Could it be updated to just expect the correct response instead?

Oh wait, the reason that I removed this is because with my new changes, a new team lead can be a member of the team (at least, the backend code won't throw an error). Therefore, my thought process was that this test endpoint isn't necessary. If it is actually necessary, how would I go about writing that test function?

Reply:
Could you change it so that instead of checking if the endpoint throws an error when a member is set to head, it checks that the member was actually removed from the members list.

Reply:
The way you could do that is by calling the setTeamHead endpoint and saving the result: `const res = await TeamsService.setTeamHead(...args);`, and then checking that in res the list of members doesn't include the new head

Reply:
Ohh ok. Yeah I'll try to fix that right now

Reply:
Ok so I think I'm getting close, but my current test runs with an error that says "Cannot edit the leads of an archived team", even though the justiceLeague team isn't archived. Do you have an idea of why this could be? I'll attach some screenshots here. Thanks!

Mock justiceLeague team:
<img width="640" alt="Screenshot 2024-03-24 at 10 05 51 PM" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/70610902/ee66ece3-51c5-4a8d-949f-552fb61727e1">

The error that is showing in the "test results" section:
<img width="890" alt="Screenshot 2024-03-24 at 10 06 56 PM" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/70610902/b873d898-d5fb-4a7a-b585-8e736bddb4cf">

My code for the actual test:
<img width="772" alt="Screenshot 2024-03-24 at 10 06 20 PM" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/70610902/52571a2e-e730-470c-8d6d-bee4f0e2cc3b">